src/System.Globalization.Calendars/tests/GregorianCalendar/GregorianCalendarEras.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using Xunit;
+
+namespace System.Globalization.CalendarsTests
+{
+    public class GregorianCalendarEras
+    {
+        private static readonly int[] s_ADEra = new int[]
+        {
+            1
+        };
+
+        #region Positive tests
+        // PosTest1: Get the eras of Gregorian calendar
+        [Fact]
+        public void PosTest1()
```

Comment:
It'd be nice if either this was rewritten to use Theory, e.g.
```C#
[Theory]
[InlineData(GregorianCalendarTypes.Arabic)]
[InlineData(GregorianCalendarTypes.Localized)]
...
public void PosTest(GregorianCalendarTypes calendarType)
{
    ...
}
```
or if at least the facts were renamed to reflect what they were testing, e.g.
```C#
[Fact]
public void TestArabic()
{
    PosTest(GregorianCalendarTypes.Arabic);
}
...
```